CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 🚨 *Breaking Changes*
+* `GitImport` will no longer be downloaded automatically when using Ray locally. This reverts behavior to `v0.42.0`.
+* Internal configuration environment variables have changed.
```

Comment:
**question**: the only change I can see is moving where we hardcode the env variable names in internal modules. What's the breaking change for the user?

Reply:
Could you list them (old name->new name) in the release notes?